csc_335_restaurant/src/model/Menu.java

```diff
@@ -24,9 +24,9 @@ public Menu(){
 	}
 	
 	private void processMenuData() throws FileNotFoundException {
-		InputStream in = getClass().getResourceAsStream("/MenuData.txt");
+		InputStream in = getClass().getResourceAsStream("/resources/MenuData.txt");
```

Comment:
```suggestion
		InputStream in = getClass().getResourceAsStream("/MenuData.txt");
```
other way gives me an error